mlflow/langchain/__init__.py

```diff
@@ -153,6 +155,7 @@ def save_model(
     persist_dir=None,
     example_no_conversion=False,
     model_config=None,
+    streamable=None,
```

Comment:
Could you add type hint and/or docstring for this parameter? Since default is `None`, it may be hard to know the input type.